src/structure/4C_structure_timint_impl.cpp

```diff
@@ -3176,52 +3176,60 @@ void Solid::TimIntImpl::cmt_linear_solve()
+            Teuchos::rcp_dynamic_cast<Core::LinAlg::BlockSparseMatrixBase>(
+                Teuchos::RCP(raw_block_mat, false));
+        if (block_mat_blocked_operator == Teuchos::null)
+          FOUR_C_THROW("Matrix is not a BlockSparseMatrix");
+        Teuchos::RCP<Xpetra::CrsMatrix<SC, LO, GO, NO>> block_mat_x_crs_A22 =
+            Teuchos::make_rcp<EpetraCrsMatrix>(
+                Teuchos::rcp(block_mat_blocked_operator->matrix(1, 1).epetra_matrix()));
+
+        std::vector<size_t> striding_info_dual;
+        striding_info_dual.push_back(dim_nullspace);
+        Teuchos::RCP<Xpetra::StridedMap<LO, GO, NO>> strided_range_map_dual =
+            Teuchos::make_rcp<Xpetra::StridedMap<LO, GO, NO>>(block_mat_x_crs_A22->getRowMap(),
+                striding_info_dual, block_mat_x_crs_A22->getRowMap()->getIndexBase(), -1, 0);
+
+        Teuchos::RCP<Xpetra::MultiVector<SC, LO, GO, NO>> nullspace = Teuchos::null;
+        nullspace = Xpetra::MultiVectorFactory<SC, LO, GO, NO>::Build(
+            strided_range_map_dual, dim_nullspace);
+        for (int i = 0; i < dim_nullspace; ++i)
         {
-          Teuchos::ArrayRCP<SC> nsValues22 = nullspace22->getDataNonConst(i);
-          int numBlocks = nsValues22.size() / dimNS2;
-          for (int j = 0; j < numBlocks; ++j) {
-            nsValues22[j * dimNS2 + i] = 1.0;
+          Teuchos::ArrayRCP<SC> nullspace_values = nullspace->getDataNonConst(i);
+          int num_blocks = nullspace_values.size() / dim_nullspace;
+          for (int j = 0; j < num_blocks; ++j)
+          {
+            nullspace_values[j * dim_nullspace + i] = 1.0;
           }
         }
 
-        auto nullspace22_xEpetra = Teuchos::rcp_dynamic_cast<Xpetra::EpetraMultiVectorT < GO, NO>>(
-                nullspace22, true);
-        Teuchos::RCP<Epetra_MultiVector> nullspace22_epetra = nullspace22_xEpetra->getEpetra_MultiVector();
-        Epetra_MultiVector &nullspace22_raw = *nullspace22_epetra;
-        std::shared_ptr <Core::LinAlg::MultiVector<double>> nullspace22_core =
-                std::make_shared<Core::LinAlg::MultiVector<double>>(nullspace22_raw);
-
-        contactsolver_->params().sublist("Inverse2").sublist("MueLu Parameters").set(
-                "nullspace", nullspace22_core);
-        contactsolver_->params().sublist("Inverse2").sublist("MueLu Parameters").set(
-                "null space: vectors", nullspace22_core);
+        auto nullspace_x_epetra =
+            Teuchos::rcp_dynamic_cast<Xpetra::EpetraMultiVectorT<GO, NO>>(nullspace, true);
+        Teuchos::RCP<Epetra_MultiVector> nullspace_epetra =
+            nullspace_x_epetra->getEpetra_MultiVector();
+        Epetra_MultiVector& nullspace_raw = *nullspace_epetra;
+        std::shared_ptr<Core::LinAlg::MultiVector<double>> nullspace_core =
+            std::make_shared<Core::LinAlg::MultiVector<double>>(nullspace_raw);
+
+        contactsolver_->params()
```

Comment:
@mayrmt the `utilities/set_up_dev_env.sh` script or clang formatting seems to have turned

``contactsolver_->params().sublist("Inverse2").sublist("MueLu Parameters").set(``
``                "nullspace", nullspace22_core);``

into

``contactsolver_->params()``
``            .sublist("Inverse2")``
``            .sublist("MueLu Parameters")``
``            .set("nullspace", nullspace_core);``

Is that how it should be?